docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -384,6 +384,9 @@ Each Alluxio master writes journal to its own journal volume requested by `volum
 set of workers.
 The journal is in a shared UFS location. In this template we use HDFS as the UFS.
 
+> Note: Please make sure your Alluxio tarball contains these sample templates. 
```

Comment:
This is the only place I can think of, that this change affects.

Reply:
The tarball is not the location that needs to contain the templates, it is the docker image. I would remove this not altogether as the user should not care of this as long as we test our publish pipeline.

Reply:
Gotcha. This note is removed.